bpcosmo/pm.py

```diff
@@ -0,0 +1,145 @@
+# This module contains utility functions to run a forward simulation
+import jax
+import jax.numpy as jnp
+
+import jax_cosmo as jc
+
+import numpyro
+import numpyro.distributions as dist
+
+from jax.experimental.ode import odeint
+from jaxpm.pm import lpt, make_ode_fn
+from jaxpm.kernels import fftk
+from jaxpm.lensing import density_plane
+import haiku as hk
+
+from jaxpm.painting import cic_paint, cic_read
+from jaxpm.pm import lpt
+from jaxpm.kernels import fftk, gradient_kernel, laplace_kernel, longrange_kernel
```

Comment:
Remove duplicate imports
```suggestion
from jaxpm.kernels import gradient_kernel, laplace_kernel, longrange_kernel
```